src/compiler/crystal/codegen/llvm_builder_helper.cr

```diff
@@ -76,7 +76,35 @@ module Crystal
       builder.inbounds_gep ptr, index0, index1, name
     end
 
-    delegate ptr2int, int2ptr, and, or, not, call, bit_cast,
+    def call(func, name : String = "")
```

Comment:
Because there's too many places where codegen generates a call outside of `codegen_call_or_invoke`. I originally had it in `codegen_call_or_invoke` but I didnt feel like duplicating funclet code in the 50 other places that generate calls (think codegen for constants, there's way more though)